src/shogun/classifier/mkl/MKL.cpp

```diff
@@ -262,6 +278,24 @@ CMKL::~CMKL()
 	SG_UNREF(svm);
 }
 
+CSGObject* CMKL::clone()
+{
+	CMKL* cloned = (CMKL*) CSGObject::clone();
+#ifdef USE_GLPK
+	// GLPK params
+	cloned->self->lp_glpk_parm = SG_MALLOC(glp_smcp, 1);
+	*cloned->self->lp_glpk_parm = *self->lp_glpk_parm;
+	// GLPK problem
+	cloned->self->lp_glpk = glp_create_prob();
+	glp_copy_prob(cloned->self->lp_glpk, self->lp_glpk, GLP_ON);
+#endif
+#ifdef USE_CPLEX
```

Comment:
this can have its own issue .... 
We dont have CPLEX anywhere so we cannot actually check. Would be good to update this at some point ....